spec/services/decorators/v6/assessment_decorator_spec.rb

```diff
@@ -55,12 +55,8 @@ module V6
       end
 
       describe "#as_json" do
-        subject(:decorator) do
-          described_class.new(assessment: assessment.reload, calculation_output:,
-                              applicant:,
-                              partner:,
-                              eligibility_result:,
-                              version:).as_json
+        let(:remarks) do
+          assessment.transform_remarks([Data.define(:type, :issue, :ids).new(:other_income_payment, :unknown_frequency, %w[abc def])])
```

Comment:
Maybe this test is a clue that we should be doing the remarks transformation in the decorator?